manifests/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -185,6 +186,44 @@ spec:
           - name: istiod
             mountPath: /var/lib/istio/local
             readOnly: true
+
+        # CoreDNS sidecar. Ports are used internally, to run as non-root.
+        # This is a short-term solution - the code in istiod can also be used
+        # directly. The plan is to move coreDNS on the agent.
+        - name: dns
+          image: coredns/coredns:1.1.2
+          imagePullPolicy: IfNotPresent
+          args: [ "-conf", "/var/lib/istio/coredns/Corefile" ]
+          securityContext:
+            runAsUser: 1337
+            runAsGroup: 1337
+            runAsNonRoot: true
+            capabilities:
```

Comment:
No, it doesn't - the k8s Service is on 53 UDP and TCP 853 ( the standard ports ), with targetPort 15053. 

It seems to be working.

Reply:
ok if seems to be working, it may be a CRI-specific capabilities feature. Just leaving a note for future PR developers that if they run into binding problems with coredns, this is where to look :)